jkube-kit/jkube-kit-vertx/src/test/java/org/eclipse/jkube/vertx/generator/VertxGeneratorTest.java

```diff
@@ -92,6 +92,21 @@ public void isApplicableWithPlugin() {
     assertThat(result).isTrue();
   }
 
+  @Test
```

Comment:
isn't it duplicating the test above ?

Reply:
You're right. all isApplicable tests seem to be testing the same scenario. Shall I convert this into a parameterized test?

Reply:
yep may be better for readability